### PR TITLE
dotnet 5 thru 7

### PR DESCRIPTION
**Pull request by axtens**
Added two folders to test dotnet5 and dotnet6 projects.
Adapted Dockerfile to use alpine and use dotnet-install.sh install both dotnet 5.0 and 6.0 support

### CODE REVIEW

**Comment by axtens** [comment]
Fixed one error. Now another.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by axtens** [comment]
One of the tests is failing
```
example-syntax-error: testing...
example-syntax-error: done
example-syntax-error: comparing results.json to expected_results.json
--- /opt/test-runner/tests/example-syntax-error/results.json
+++ /opt/test-runner/tests/example-syntax-error/expected_results.json
@@ -1,4 +1,5 @@
 {
   "version": 1,
-  "status": "pass"
+  "status": "fail",
+  "message": "/solution/Leap.vb(1,18): error BC30205: End of statement expected. [/solution/Leap.vbproj]"
 }
```
but when I run it interactively, it works fine.

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ RUN mkdir -p /usr/share/dotnet \
     chmod +x dotnet-install.sh && \
     ./dotnet-install.sh -c 5.0 --install-dir /usr/share/dotnet && \
     ./dotnet-install.sh -c 6.0 --install-dir /usr/share/dotnet && \
+    ./dotnet-install.sh -c 6.0 --install-dir /usr/share/dotnet && \
```
Comment (ErikSchierboom) [inline Dockerfile:12]: I think you want:

```suggestion
    ./dotnet-install.sh -c 7.0 --install-dir /usr/share/dotnet && \
```

File: tests/example-success-7/Leap.vbproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>    
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (ErikSchierboom) [inline tests/example-success-7/Leap.vbproj:4]: This should be:

```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

**Comment by axtens** [comment]
Still got failing checks

**Comment by ErikSchierboom** [comment]
Unfortunately, this does not pass the tests: https://github.com/exercism/vbnet-test-runner/actions/runs/3882376102/jobs/6622455326

**Comment by axtens** [comment]
Which I don't understand because the Leap.vb file contains only
```
Public Moduleeee@23 Leap
```
which in every other environment throws the error that the `expected_results.json` file contains. Cacheing error?

**Comment by axtens** [comment]
Or is there a way of running the CD/CI with verbose logging?

**Comment by ErikSchierboom** [comment]
I don't think verbose logging would give you much, as it the test runner output is shown already.

**Comment by axtens** [comment]
I'm confused

Interactively it fails appropriately.  
```
C:\Users\bugma\Dropbox\Projects\test-runners\vbnet-test-runner\tests\example-syntax-error>dotnet test
  Determining projects to restore...
  Restored C:\Users\bugma\Dropbox\Projects\test-runners\vbnet-test-runner\tests\example-syntax-error\Leap.vbproj (in 7.
  16 sec).
C:\Users\bugma\Dropbox\Projects\test-runners\vbnet-test-runner\tests\example-syntax-error\Leap.vb(1,18): error BC30205:
 End of statement expected. [C:\Users\bugma\Dropbox\Projects\test-runners\vbnet-test-runner\tests\example-syntax-error\
Leap.vbproj]
```
During CD/CI it fails INappropriately.
```
 example-syntax-error: testing...
example-syntax-error: done
example-syntax-error: comparing results.json to expected_results.json
--- /opt/test-runner/tests/example-syntax-error/results.json
+++ /opt/test-runner/tests/example-syntax-error/expected_results.json
@@ -1,4 +1,5 @@
 {
   "version": 1,
-  "status": "pass"
+  "status": "fail",
+  "message": "/solution/Leap.vb(1,18): error BC30205: End of statement expected. [/solution/Leap.vbproj]"
 }
Error: Process completed with exit code 1.
```

**Comment by ErikSchierboom** [comment]
I fixed it in https://github.com/exercism/vbnet-test-runner/pull/26/commits/c281367fad9ab59ffeb3ebe5a4b1b8318a0ab0de

**Pull request merged**